blockware/pixel-dust/src/main.cpp

```diff
@@ -262,7 +265,10 @@ void setup(void) {
 
   if(!sand.begin()) err(1000); // Slow blink = malloc error
 
-  if(!accel.begin(Wire, ACCEL_ADDR)) err(250);  // Fast bink = I2C error
+  acc = new LIS2DW12Sensor(&Wire, 0x31U);
```

Comment:
Can you only call this after `Wire.begin()` and that's why you have to allocate it to the heap?